newshomepages/utils.py

```diff
@@ -643,3 +643,10 @@ def _load_new_page_disable_javascript(
     print(f"Waiting {wait_seconds} seconds")
     time.sleep(wait_seconds)
     return page
+
+
+def read_script_from_file(filename: str, page_obj: Page):
```

Comment:
Does this need to be broken out into a function? Are we using it more than once?